src/main/java/com/liferay/ide/idea/core/LiferayPostStartupActivity.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.idea.core;
+
+import com.intellij.openapi.externalSystem.service.project.manage.ProjectDataImportListener;
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.project.DumbAware;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.startup.StartupActivity;
+import com.intellij.openapi.startup.StartupManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.util.messages.MessageBus;
+import com.intellij.util.messages.MessageBusConnection;
+
+import com.liferay.ide.idea.util.FileUtil;
+import com.liferay.ide.idea.util.IntellijUtil;
+
+import java.util.stream.Stream;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.idea.maven.project.MavenImportListener;
+
+/**
+ * @author Simon Jiang
+ */
+public class LiferayPostStartupActivity implements StartupActivity, DumbAware {
+
+	@Override
+	public void runActivity(@NotNull Project project) {
+		VirtualFile projectDir = project.getBaseDir();
+
+		projectDir.refresh(false, true);
+
+		VirtualFile directoryToExclude = projectDir.findChild("bundles");
```

Comment:
In my opinion we should not use "bundles" always. Depending on how you configure e.g. the maven bundle-support plugin the directory may be a different one. So better read the configuration of the maven or gradle plugin and determine the correct directory.

Reply:
yes, We found this issue and will change it soon. Thanks